src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html

```diff
@@ -57,9 +57,16 @@ <h4 id="description">click on slots to change availability</h4>
                 </div>
 
                 <div class="col-lg-1">
-                    <button class="btn btn-primary" type="button" ng-model="editNote" ng-click="editNoteSwitch()" ng-init="editNote = false; buttonTitle = 'Edit note'">{{buttonTitle}}</button>
+                    <div class="btn-group-vertical">
+                        <button class="btn btn-primary" type="button" ng-model="editNote" ng-click="editNoteSwitch()" ng-init="editNote = false; buttonTitle = 'Edit note'">{{buttonTitle}}</button>
```

Comment:
Inline code in attributes should be avoided. It would be a good idea to move the content of `ng-init` into an appropriate function.